examples/lark_grammar.py

```diff
@@ -14,6 +14,7 @@
 
 grammar_files = [
     examples_path / 'lark.lark',
+    examples_path / 'no_newline_at_end.lark',
```

Comment:
This is not the right path for it. Either put it in `tests/` or as an internal string in this file.